### DISCUSSION TITLE
Telegram Help

### DISCUSSION THREAD

**Discussion by hadeedsarfaraz** [jef/streetmerchant#2395] [Help]
I Have set up streetmerchant, but I'm curious if anyone has a step by step guide to do telegram notifications, I have a single bot in 2 groups with chat id pulled from telegram bot raw, and the authentication token from botFather but I'm not sure if that's all I need.

**Comment by elebertus** [comment]
```shell
TELEGRAM_ACCESS_TOKEN='\d{10}:([A-z]\|\d){35}'
TELEGRAM_CHAT_ID='(?:-)\d{13}'
```
Should just be these two vars in your `dotenv`, pretty sure those regexes are correct.. I had issues with this because the ID contained a `-` prefix which I wasn't including. I was personally new to telegram so I wasn't sure (and am still not sure why a `-` is in the ID).

Not 100% sure if you're asking if there is a way to wire in multiple groups through the config or not. If you are I'm not quite sure but the docs at least don't mention that.
